CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [4.3.1] - 2019-12-23
 
+### Added
+
+- Configure routing timeout in edxapp application.
+  routes and all nginx configurations share the same setting: edxapp_routing_timeout
```

Comment:
This is too verbose for a CHANGELOG IMO.
```suggestion

```

Reply:
I decided to be in verbose mode this year.